deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -643,6 +643,30 @@ spec:
                                               type: string
                                             mode:
                                               type: integer
+                  resources:
+                    type: object
+                    description: Specifies the compute resource requirements for the storage pod.
```

Comment:
guess you just copied it from storage pod resource setting? need to rephrase those descriptions.

Reply:
nice catch.